webapp/src/containers/Rhs/rhs.scss

```
@@ -114,6 +114,11 @@
   width: 88px;
   height: 40px;
   border-radius: 4px;
+
+  &__without-filter {
+    position: absolute;
+    right: 0;
+  }
```

```suggestion
  &__without-filter {
    margin-left: auto;
  }
```

on line 42 we have `div className='d-flex align-item-center'` which means we have a flex container so to align any element inside a flex container to the rightmost you can use `margin-left: auto`